src/main/java/duke/command/UntickCommand.java

```diff
@@ -0,0 +1,24 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.Ui;
+import duke.Storage;
+import duke.task.Task;
+
+public class UntickCommand extends Command{
+    private int taskIndex;
+
+    public UntickCommand(int taskIndex) {
+        this.taskIndex = taskIndex;
+    }
+    @Override
```

Comment:
```suggestion
    }
    
    @Override
```
Whitespace here too! :)